### ISSUE TITLE
Decide if users messing with metadata warrants ERROR condition

### ISSUE THREAD

**Issue by lvh** [rackerlabs/otter#981]
[Original ask](https://github.com/rackerlabs/otter/pull/978/files#r23713580).

Basically: if the user messes with metadata, do we silently ignore the metadata, or do we actively put the group in an error state?

**Comment by wirehead** [comment]
No, we should silently do the 'right' thing, but we should log a "user edited metadata" message to cloud feeds and kibana.

**Comment by lvh** [comment]
But what is the right thing? Ignoring the metadata? There's reasons we're putting it there.

**Comment by wirehead** [comment]
We should do something that is consistent, easy to explain, and not especially hard to implement.

For example, we discussed the LB case and I'm OK with just removing the server from all LBs (I'm also OK with not removing it from any LBs if that will make the code significantly easier).

Given that we're going to make this hard to edit in other systems over time and that we can document our keys and tell the user to just not do it, we shouldn't see too many users messing with the metadata.

But it seems like overkill to actually throw the user into an error state.

**Comment by radix** [comment]
I recommend we just log a cloud feeds log message in the code under discussion, and leave the group ACTIVE.

**Comment by manishtomar** [comment]
Yeah, if we can detect it then a cloud feeds message should do the trick. If we cannot detect it then well... nothing can be done :)

**Comment by radix** [comment]
@manishtomar the link in the description is to a specific place where we're catching errors from messed up metadata

**Comment by manishtomar** [comment]
@radix Sorry yes I realized that. However, this ticket's description seemed to be generic enough to respond that way :)
